ci-operator/step-registry/upi/conf/gcp/xpn/upi-conf-gcp-xpn-commands.sh

```diff
@@ -24,15 +24,22 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-HOST_PROJECT="openshift-dev-installer"
-CLUSTER_NETWORK="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/global/networks/installer-shared-vpc"
-CONTROL_SUBNET="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/regions/us-east1/subnetworks/installer-shared-vpc-subnet-1"
-COMPUTE_SUBNET="https://www.googleapis.com/compute/v1/projects/openshift-dev-installer/regions/us-east1/subnetworks/installer-shared-vpc-subnet-2"
-COMPUTE_SERVICE_ACCOUNT="do-not-delete-ci-xpn@openshift-gce-devel-ci.iam.gserviceaccount.com"
-CONTROL_SERVICE_ACCOUNT="do-not-delete-ci-xpn@openshift-gce-devel-ci.iam.gserviceaccount.com"
-PRIVATE_ZONE_NAME="ci-op-xpn-private-zone"
+if [[ -f "${CLUSTER_PROFILE_DIR}/xpn_project_setting.json" ]]; then
```

Comment:
Sorry I'm lost, where is this `xpn_project_setting.json` coming from?

Reply:
It's from QE's [vault](https://vault.ci.openshift.org/ui/vault/secrets/kv/show/selfservice/openshift-qe/gcp-qe), where QE's cloud credentials are stored.

Reply:
Ok. If we further developer the XPN workflow these settings may be likely to change because some of them--at least the networks--are region dependent so they should probably be attached to the boskos leases.

That is not an issue with this PR, just mentioning that this may change in the future.